storage/postgres/platform.go

```diff
@@ -42,6 +42,7 @@ type Platform struct {
 	CredentialsActive bool           `db:"credentials_active"`
 	LastActive        time.Time      `db:"last_active"`
 	Technical         bool           `db:"technical"`
+	Version           string 	     `db:"version"`
```

Comment:
indentation (or does it only look like this in the browser?)

Reply:
there a few more indentation "issues", can you `go fmt` everything?

Reply:
only in the browser